app/src/timelinecells.cpp

```diff
@@ -784,16 +783,17 @@ void TimeLineCells::paintEvent(QPaintEvent*)
             painter.restore();
 
             painter.setPen(palette.color(QPalette::HighlightedText));
-            int incr = (mEditor->currentFrame() < 10) ? 4 : 0;
+            int incr = (currentFrame < 10) ? 4 : 0;
             painter.drawText(QPoint(currentFrameStartX + incr, 15),
-                             QString::number(mEditor->currentFrame()));
+                             QString::number(currentFrame));
         }
     }
 }
 
 void TimeLineCells::resizeEvent(QResizeEvent* event)
 {
-    clearCache();
+    delete mCache;
```

Comment:
imo. I think it's better to keep the clearCache method, having to call delete and set it to nullptr manually seems error prone.